### PR TITLE
Update changelog post sprint 22

### PR DESCRIPTION
**Pull request by PVince81**
@pmaier1 FYI

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,9 +59,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Removed
 
 - Removed bundled documentation, help links now point to the online documentation - [#34612](https://github.com/owncloud/core/issues/34612) [#34649](https://github.com/owncloud/core/issues/34649)
+- Remove incompatible script for generating DB changeset - [#34722](https://github.com/owncloud/core/issues/34722)
 
 ### Fixed
 
+- Dont't expose hashed password in OCS api - [#34691](https://github.com/owncloud/core/issues/34691)
```
Comment (phil-davis) [inline CHANGELOG.md]: s/Dont't/Don't/

**Comment by phil-davis** [comment]
Also this needs to be rebased. I made a recent change to drone pipeline/service names like `webdav-apache` and the CI seems to have picked up only part of it - some phpunit jobs failed.

**Comment by PVince81** [comment]
I don't want to rebase as I'd need to also add newer changelog entries from the update.

File: CHANGELOG.md (hunk 1 above)
Comment (PVince81) [reply CHANGELOG.md]: fixed

**Pull request merged**